packages/ckb-sdk-core/examples/sendTransactionWithMultiplePrivateKey.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable */
+const Core = require('../lib').default
+const core = new Core('http://localhost:8114')
+
+const sk1 = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' // exmaple private key
+const sk2 = '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff' // example private key
+
+const loadCells = async () => {
+  await core.loadSecp256k1Dep()
+  const lockScript1 = {
+    codeHash: core.config.secp256k1Dep.codeHash,
+    hashType: core.config.secp256k1Dep.hashType,
+    args: `0x${core.utils.blake160(core.utils.privateKeyToPublicKey(sk1), 'hex')}`
+  }
+
+  const lockScript2 = {
```

Comment:
After executing`await core.loadSecp256k1Dep()`,  fetching code hash and hash type from config is strange.

Reply:
The user can use the returned value of `core.loadSecp256k1Deps()` or the cached value of `core.config.secp256k1Dep`

Reply:
OK. Maybe I will use the returned value of `core.loadSecp256k1Deps()` directly, because I think it is more clear.